cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+- Perform issue triage on and maintain the spinnaker/contributor-experience repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Document and assist SIGs/WG Chairs and Technical Leads with organizational management operations
+- Distribute contributor related news on various Spinnaker channels, including Continous Delivery Foundation ([CDF](https://cd.foundation)) for posting blogs, social media, and other platforms as needed.
+- Establish and share metrics to measure project health, community health, and general trends, including:
+  - [DevStats](https://spinnaker.devstats.cd.foundation/)
+  - the CDF contribution gamification app [[WIP](https://github.com/ExitoLab/spinnaker_gamification_app)]
+  - contributor experience survey(s)
+  - engagement on project platforms
+  - pull request and issue engagement and closure times
+- Set community health goals such as improved PR and issue engagement and closure times, and track progress towards their achievement using established metrics.
+- Research other OSS projects and consult with their leaders about contributor experience topics to make sure we are always delivering value to our contributors
+- Create and maintain subprojects that incentivize community outreach, individual content creation, and welcoming/onboarding of new contributors
+
+#### Cross-project and Externally Facing Processes
+
+We cross-cut all SIGs and working groups to deliver the following processes:
+
+- Deploying Changes:
+ This SIG will create change-deployment policies that balance quick response to the needs of the community and a limit on disruptions for project contributors. We will create policies to socialize and elicit feedback on changes via community channels such as Slack, spinnaker.io, Twitter, mailing lists, SIG meetings, delegating to SIG leads, etc. The amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We can estimate risk based on these parameters:
+
+- Low-risk changes impact a small number (<3) of SIGs, working groups, or repositories, do not break existing contributor workflows, and are easy to roll back.
+- High-risk changes impact a large number (>3) of SIGs, working groups, or repositories, break existing contributor workflows and are not easy to roll back.
+- Depending on how wide of an ecosystem change this is, we may also slack, blog, tweet, and use other channels to get the word out.
+- Our standard time box is 72 business hours; however, there may be situations where we need to act quickly but the time period will always be clear and upfront.
```

Comment:
72 business hours is 9 business days. Is this language intentional?

Regardless, I suspect using "business days" is going to lead to problems: Business days are locale-specific, which the Spinnaker organization is not (Canada does not have the same business holidays as the United States, for example).

Reply:
We may need to revisit this time box

Reply:
I think this time box should be revisted after the SIG is founded and determines what the capacity is for response. I recommend deleting this and adding that determination as a goal @ExitoLab @Prophen